src/Services/src/CfCli/ICfCliService.cs

```diff
@@ -28,5 +28,6 @@ public interface ICfCliService
         Task<DetailedResult> PushAppAsync(string appName, StdOutDelegate stdOutCallback, StdErrDelegate stdErrCallback, string appDir, string buildpack = null, string stack = null);
         Task<Version> GetApiVersion();
         Task<DetailedResult<string>> GetRecentAppLogs(string appName);
+        void ClearCachedAccessToken();
```

Comment:
Caching is a general purpose feature which may be useful elsewhere.

How about defining a new interface with a single method:
```
interface ICacheable
{
    ClearCache();   // or ClearCaches();
}
```
and having ICfCliService extend the ICacheable interface.

Reply:
Ok yeah, could do! Are you thinking `ClearCache` would eliminate all cached values on a class that implements `ICacheable`? I know there's only 1 cached values at the moment (access token), but in general I like the option to have fine-tuned control over which cached values to reset